liferay-frontend/liferay-portal
# LPS-162554 Miss language key for staging error message

Motivation
=======
This work solves the problem reported in the [LPS-162554](https://issues.liferay.com/browse/LPS-162554) ticket.
The solution was passed my review. PR: TaiPhamLR#3
cc: @sontruongces

Proposed Solution
========
What we did to solve this problem.
1. Add key-value to language properties

Steps to Verify
========
1. Create a new site and enable local staging with all STAGED CONTENT.
2. Create a content page and add a Banner fragment to the page.
3. In the Banner fragment, add the Background Image via mapping it from the Styles Panel.
4. Publish the content page.
5. Create a web content entry.
6. Create another content page and add a WCD widget to the page.
7. Configure the WCD widget to show the previous added web content entry.
8. Publish the content page.
9. Go to Publishing -> Staging
10. Trigger a custom publication with the following settings:
    1. Select all pages
    2. Uncheck web content entry
11. Go to live site and delete the published image file under DM admin.
12. Navigate to staging site page and click on Publish to Live button.
13. In the Publish to Live page click on 'Switch to Advance Publish Process' link in the top-right.
14. Publish to live with the default settings.
15. Click on the See More Details button after the publication process is failed.

- Expected Results: Proper language key should be set for the error message.
- Actual Results: The error message says "consider-that-the-following-data-would-not-have-been-published-either".
